armi/bookkeeping/report/reportInterface.py

```diff
@@ -94,6 +95,54 @@ def interactEveryNode(self, cycle, node):
             else:
                 runLog.warning("No mgFlux to plot in reports")
 
+        self.printParameterTable()
+
+    def printParameterTable(self):
+        nodeParameters = [
+            self.r.p.cycle,
+            self.r.p.timeNode,
+            self.r.core.p.coupledIteration,
+            self.r.core.p.keffUnc,
+            self.r.core.p.THdeltaPCore / 1e6,
+            self.r.core.getMaxBlockParam("percentBuPeak"),
+            self.r.core.getMaxBlockParam("puFrac"),
+            self.r.core.getMaxParam("THmassFlowRate"),
+        ]
+        paramNames = [
```

Comment:
You can get the "description" field from a parameter. So you don't have to maintain this list manually.

Reply:
It might be good to implement this type of summary on the application level rather than in ARMI since different applications may care about different or arbitrary parameters.

Reply:
Actually, Jake makes a good point. I hadn't thought of that, but yeah totally.